astropy/utils/masked/core.py

```diff
@@ -1078,10 +1080,18 @@ def argsort(self, axis=-1, kind=None, order=None):
 
         return np.lexsort(keys, axis=axis)
 
-    def sort(self, axis=-1, kind=None, order=None):
-        """Sort an array in-place. Refer to `numpy.sort` for full documentation."""
+    def sort(self, axis=-1, kind=None, order=None, *, stable=False):
+        """Sort an array in-place. Refer to `numpy.sort` for full documentation.
```

Comment:
Here, too, we could write two versions of the methods, or go for `*args, **kwargs`

Reply:
For now I've duplicated the implementation. Let me know if that is satisfactory.